src/main/java/org/sagebionetworks/bridge/models/TimestampHolder.java

```diff
@@ -0,0 +1,19 @@
+package org.sagebionetworks.bridge.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class TimestampHolder {
```

Comment:
Usually in our APIs, time values are expressed using ISO 8601 string values... where we store longs in the persistence tier, these values are always in UTC ("Z") time, but are still immediately recognizable in the API as time values. I would personally prefer we return something like a DateTimeHolder than a LongHolder (which is all this is).